File: C/unit_5/Bb.c
#include<stdio.h>
int main()
{
  int num,fact = 1,i=1;
  printf("Enter the Number : \n");
  scanf("%d",&num);
  do
  {
      fact*=i;
      i++;
  } while (i<=num);
  printf("The factorial of %d is : %d \n",num,fact);
  return 0;
}

File: Java/unit_5/Bb.java
package Java.unit_5;

import java.util.Scanner;

public class Bb {
    public static void main(String[] args) {
        int num, fact = 1, i = 1;
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter the Number : \n");
        num = sc.nextInt();
        do {
            fact *= i;
            i++;
        } while (i <= num);
        System.out.println("The Factorial of " + num + " is : " + fact);
        sc.close();
    }
}
